docs/details/commands/pmm-admin.md

```diff
@@ -323,6 +323,59 @@ When you remove a service, collected data remains on PMM Server for the specifie
         - `push`: agent will push metrics.
         - `pull`: server scrapes metrics from agent.
 
+## Advanced Options
+
+### Enable all collectors
+
+PMM starts the MongoDB exporter by default by using these these parameters:
+
+|Parameter|Description|
+|-----|-----|
+|`--collector.diagnosticdata`|Enables `getDiagnosticData` collector|
+|`--collector.replicasetstatus`|Enables `replSetGetStatus` collector|
+|`--discovering-mode`|Auto discovers all databases and collections|
+|`--compatible-mode`|Enables MongoDB exporter v1 compatible metric names|
+|`--mongodb.global-conn-pool`|Uses a single connection to the DB instead of connecting in each scrape|
```

Comment:
I've realized this description is confusing. I've read it as "this is pmm-admin flags,"  but in reality  - this is exporters flag, and we'd better mention this is Exporter  and send people to  Exporter  documentation  to read more about  flags and what they'll collect